plugins/chat/assets/javascripts/discourse/components/chat-channel.hbs

```diff
@@ -68,14 +68,14 @@
   />
 
   {{#if this.pane.selectingMessages}}
-    <ChatSelectionManager
-      @selectedMessageIds={{this.pane.selectedMessageIds}}
-      @chatChannel={{@channel}}
-      @cancelSelecting={{action
-        this.pane.cancelSelecting
-        @channel.selectedMessages
+    {{@channel.isDirectMessageChannel}}
```

Comment:
I guess this is just leftover debugging stuff?

Reply:
yes :p I want to say I would have removed it today after looking at it before moving off from draft... but who knows.